include/wx/sizer.h

```diff
@@ -669,6 +669,9 @@ class WXDLLIMPEXP_CORE wxSizer: public wxObject, public wxClientDataContainer
     // this size to really update all the sizer items.
     virtual wxSize CalcMin() = 0;
 
+    // Can be overidden for two step process, e.g. by wxWrapSizer
+    virtual wxSize CalcMinFirstPass() { return CalcMin(); }
```

Comment:
It would be nice to explain what the two step process is, because this is really not obvious.

Also, `wxWrapSizer` doesn't really _need_ it, as it worked before adding it, so this makes things even more confusing.

Reply:
`wxWrapSizer` was, AFAIR, somewhat of a hack and it's great to replace it with something better but this something should be documented/explained.

Maybe documenting it would also give some idea for a better name because, frankly, "first pass" is not great: it doesn't mean anything per se and implies the existence of "second pass" which doesn't actually exist.